### DISCUSSION TITLE
Why no routing on TopMetal 1&2 ?

### DISCUSSION THREAD

**Discussion by maliberty** [IHP-GmbH/IHP-Open-PDK#282] [Q&A]
@KrzysztofHerman in https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/blob/7e8900be6942b8f67e67211f84a24089fde3b690/flow/platforms/ihp-sg13g2/config.mk#L94 you set the max routing layer to Metal5.  I'm wondering why you excluded TopMetal 1&2?  There is a fair amount of routing resource on those layers even with pdn stripes.

**Comment by KrzysztofHerman** [comment]
@maliberty `TopMetal1` and `TopMetal2` are thick (3 um) aluminum metals typically used for power grid and for redistribution layer routing. Anyway if someone needs these layers for routing it can be easily changed in the configuration.

**Reply by maliberty** [reply]
Do you use bumps for packing at IHP?  I was aware of the need for RDL.  The power grid doesn't use all the resources and routers can avoid it.
